prime-router/src/main/kotlin/history/azure/DeliveryFunction.kt

```diff
@@ -203,10 +209,108 @@ class DeliveryFunction(
         }
     }
 
+    /**
+     * Get a sortable list of facilities for multiple deliveries
+     *
+     * @param request HTML request body.
+     * @return JSON of the facility list or errors.
+     */
+    @FunctionName("getBulkDeliveryFacilities")
+    fun getBulkDeliveryFacilities(
+        @HttpTrigger(
+            name = "getBulkDeliveryFacilities",
+            methods = [HttpMethod.GET],
+            authLevel = AuthorizationLevel.ANONYMOUS,
+            route = "waters/report/{receiver}/facilities"
```

Comment:
I think we can punt on this for now since it's unclear whether the other dashboard views are org or receiver based, but I think receiver might ultimately make sense as a filter?

Reply:
Let's go with `orgService` instead of `receiver` here since those are the names used in a lot of existing code. We should only use `receiver` when we have a formalized definition of it